src/recipe/responses/recipe.response.ts

```diff
@@ -8,17 +9,21 @@ import { CreatePreparingDTO } from '../DTOs/create-preparation.dto';
 import { CreateIngredientDTO } from '../DTOs/create-ingredient.dto';
 import { IngredientResponse } from './ingredient.response';
 import { ApiProperty } from '@nestjs/swagger';
+import { ImageResponse } from '../responses/image.response';
+import { CreateImageDTO } from '../DTOs/create-image.dto';
 
 export class RecipeResponse implements Recipe {
   constructor(
     recipe: Recipe & { preparing: RecipePreparationSteps[] } & {
       ingredients: RecipeIngredients[];
-    },
+    } & { image?: Image },
```

Comment:
Please refer to prisma typing documentation. I think there is a better way to type the responses from db.
Please check if prisma allows you to reuse prismas generated types to describe joined properties